selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -273,7 +273,7 @@ void DevicePanel::updateCalibDescription() {
       AlignedBuffer aligned_buf;
       capnp::FlatArrayMessageReader cmsg(aligned_buf.align(calib_bytes.data(), calib_bytes.size()));
       auto calib = cmsg.getRoot<cereal::Event>().getLiveCalibration();
-      if (calib.getCalStatus() != 0) {
+      if (calib.getCalStatus() != cereal::LiveCalibrationData::Status::UNCALIBRATED) {
```

Comment:
want this to show while recalibrating too?

Reply:
No that's fine, if recalibrating it already wrote new values.